README.md

```diff
@@ -34,7 +34,7 @@ The `main` branch is the bleeding edge of the v1 migration. The `master-v0.x` br
 v0 release (where we are still merging some patches and crucial fixes).
 
 The current v1 release is
-<a title="Pub" href="https://pub.dartlang.org/packages/flame" ><img src="https://img.shields.io/pub/v/flame.svg?style=popout&include_prereleases" /></a>
+<a title="Pub" href="https://pub.dev/packages/flame/versions/1.0.0-releasecandidate.13" ><img src="https://img.shields.io/pub/v/flame.svg?style=popout&include_prereleases" /></a>
```

Comment:
We should probably reference to the "release" list instead of the specific version, as we would need to keep that updated. And change the text to like "The current v1 release can be found [here](https://pub.dev/packages/flame/versions#prerelease)

```suggestion
<a title="Pub" href="https://pub.dev/packages/flame/versions#prerelease" ><img src="https://img.shields.io/pub/v/flame.svg?style=popout&include_prereleases" /></a>
```

Reply:
@wolfenrain Is the [shields.io image](https://img.shields.io/pub/v/flame.svg?style=popout&include_prereleases) manually updated? Or does it always point to the latest pre-release version?

If its manual, updating the link could be part of that manual process. If it's automatic, then the pre-release tag would be better. Although an update of the copy would need to be made as "The current v1 release is" suggests you'll go directly to the latest version!

The Flame [pub.dev page](https://pub.dev/packages/flame/versions) does it two different ways. The pre-release link underneath the title links directly to RC13, but the one above "Stable versions of flame" links to the tag as you suggested.

How do they get updated? Is there a way to automatically link to the latest release? This is probably a little too much to be thinking about a link at this stage :P My apologies!

Reply:
the tag always points to the newest one, it's auto

Reply:
*shield